code/tests/edge/test_stateStoreManager.brs

```diff
@@ -20,6 +25,138 @@ sub TC_adb_StateStoreManager_Init()
     UTF_assertEqual([], actualStateStore, generateErrorMessage("State store", "[]", actualStateStore))
 end sub
 
+' target: _adb_StateStoreManager()
+' @Test
+sub TC_adb_StateStoreManager_Init_stateStorePersisted_notExpired()
+    stateStoreManager = _adb_StateStoreManager()
+
+    ' Mock persisted state store
+    stateStoreMap = {
+        "kndctr_1234_AdobeOrg_cluster": {
+            "payload" : {
+                    key: "kndctr_1234_AdobeOrg_cluster",
+                    value: "or2",
+                    maxAge: 1800
+                },
+            "expiryTs": _adb_timestampInMillis() + 1000
```

Comment:
nit: Consider making the timeout larger than 1 second in case of slow CI systems to avoid false negative. Maybe set it to 3 or 5 seconds?

Reply:
These tests don't run on CI as there is no roku device support.